gandalf.js

```diff
@@ -4,121 +4,125 @@
  * This plugin performs initialization (like calling mermaid.initialize())
```

Comment:
SCRIPT-I'm a bit concerned about the potential performance impact of this loop. Have you thought about optimizing it?